### PR TITLE
Fix package metadata

### PR DESCRIPTION
**Pull request by Eilon**
@HaoK @divega @rustd

### CODE REVIEW

**Comment by Eilon** [comment]
Merged, but please have a look to ensure the metadata is good.

File: src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/project.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
-  "description": "ASP.NET Identity providers that use Entity Framework.",
+  "description": "ASP.NET Core Identity providers that use Entity Framework Core.",
```
Comment (divega) [inline src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/project.json]: I wonder why we were using plural "providers".  I think of this package as containing just one "provider" which contains a set of store classes.

**Comment by divega** [comment]
LGTM other than a minor comment on something that was already there. Thanks.

File: src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/project.json (hunk 1 above)
Comment (Eilon) [inline src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/project.json]: https://github.com/aspnet/Identity/commit/ebd6ed7d94e59c3d9154528888956da4b318fa93

Comment (divega) [inline src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/project.json]: "use"  - > "uses"?  :smile:

Comment (Eilon) [inline src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/project.json]: https://github.com/aspnet/Identity/commit/f4b2548565b2ae7161ae3630cc6375b0dd41e3d2 :rage: 



:smile:

Comment (Eilon) [inline src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/project.json]: I'll keep messing it up until the SHA hash spells out a word that expresses how silly I feel :smile:

**Pull request merged**